test/repl.jl

```diff
@@ -348,7 +348,7 @@ begin
     LineEdit.accept_result(s, histp)
     @test LineEdit.mode(s) == repl_mode
     @test buffercontents(LineEdit.buffer(s)) == "shell"
-    @test position(LineEdit.buffer(s)) == 4
+    @test position(LineEdit.buffer(s)) == 2
```

Comment:
I'd like to have this resolved before merging. Not because I particularly care about this use case, but because there may be something non-obvious happening that we should know about as we transition packages.